ompi/contrib/libompitrace/Makefile.am

```diff
@@ -41,3 +41,5 @@ libompitrace_la_SOURCES = \
         request_free.c \
         send.c \
         sendrecv.c
+
+libompitrace_la_LDFLAGS = -version-info 20:0:0
```

Comment:
@jsquyres  Is there a particular reason why we hard code the `20:0:0` instead of using a variable like `$(libompitrace_so_version` that would be defined in `VERSION`?

Seems like this is the only place where we hardcode the version info. All other places reference the `VERSION` file or something similar.